debian/index.js

```
@@ -76,29 +77,24 @@ const start = () => {
         exec('kill -9 $(pgrep Headset) &> /dev/null')
       }
     })
+    
+    // Code based on MarshallOfSound Google-Play-Music-Desktop-Player-UNOFFICIAL-
+    // file src/main/features/linux/mediaKeysDBus.js
+    // Other desktop environments might have their own dbus which can be added
+    try {
+      const bus = DBus.getBus('session');
+
+      registerBindings('gnome', bus);
+      registerBindings('mate', bus);
+      
+    } catch (err) {
+      
```

Ehy are we silently failing here?